stock_picking_rate/tests/test_stock_picking_rate_purchase.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from .common import TestHelper
+
+
+class TestStockPickingRate(TestHelper):
+
+    def setUp(self):
+        super(TestStockPickingRate, self).setUp()
+        self.rate_ids = [self.new_record(), self.new_record()]
+        self.Wizard = self.env['stock.picking.rate.purchase']
+        self.wizard_vals = {
+            'rate_ids': [(6, 0, [r.id for r in self.rate_ids])],
+        }
+
+    def new_wizard(self, vals=None):
+        vals = self.wizard_vals if vals is None else {}
+        return self.Wizard.create(vals)
+
+    def test_default_rate_ids(self):
+        """ It should default to active rate ids """
+        exp = [r.id for r in self.rate_ids]
+        rec_id = self.Wizard.with_context(
+            active_ids=exp,
+            active_model=self.Wizard._name,
+        ).create({})
+        res = [r.id for r in rec_id.rate_ids]
```

Comment:
@lasley when you have a chance can you take a look at this? It's returning an empty array, which is causing another test to fail.